components/AppNav.tsx

```diff
@@ -39,72 +32,52 @@ const navItems: NavItem[] = [
   }
 ]
 
-const NavBar: React.FC<AuthLinkProps> = ({ session }) => {
+const NavBar: React.FC<AuthLinkProps> = ({}) => {
   const router = useRouter()
-  const isAdmin = _.get(session, 'user.isAdmin', false) as boolean
   const location = router.asPath
+
   return (
     <>
       {navItems.map(button => (
         <NavLink
           {...button}
-          className={`${styles['nav-item']} nav-link`}
+          className={`${styles['nav-item']} ${styles['nav-link']} px-3`}
           key={button.name}
-          activePath={location === button.path}
+          activePath={location.includes(button.path)}
```

Comment:
What was wrong with the previous logic? `location === button.path`

Reply:
I used `include` to highlight the navigation link when the user is under a subpage of that path, for now the only use case is mostly for the curriculum subpages, to indicate that they are still inside the curriculum path, i believe it's great for the UX. It will also be useful later if we add submenus for the the different new features.

Reply:
Can you please list all UX changes in your PR description

Reply:
Description updated!